apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -257,15 +257,19 @@ const linkingOptions = {
               [ScreenName.ReceiveSelectCrypto]: "receive",
             },
           },
+          // FIXME: this code was getting overriden. is it the correct one?
```

Comment:
@mlegall-ledger can you check which of the 2 codes are to keep? there were twice the definition of `[NavigatorName.Swap]` and i'm not sure which one is to keep.

thanks